### ISSUE TITLE
Costume Sets not available

### ISSUE THREAD

**Issue by Dataweaver42** [Perdition-117/COM3D2.DressCode#5]
![img20230311012339](https://user-images.githubusercontent.com/21045510/224468377-9198515e-fb7d-4342-9d97-e84933c97a61.png)

**Comment by Perdition-117** [comment]
Hi!

Sorry, I have not had my notifications in order for a few days, but it just so happens that I fixed this issue today. A new release should be out shortly.

**Comment by Dataweaver42** [comment]
A broader "fix" would be to simply not disable so much of the costume editor.  I could definitely see allowing the player to use different hair styles with different costumes, for instance; or to apply makeup or tattoos for one costume but not the others. My recommendation would be to implement the full standard Editor with only two changes: disable the Profile control, and add the Clothing Setting control. But don't bother restricting anything else: if the player wants to alter aspects of Head, Hair, or Body for a particular costume, let him. If he wants to randomize aspects of the maid's appearance, let him. If he wants his maid to look radically different when dancing or doing yotogi than she looks normally, let him do that. Let _him_ be the one who imposes whatever standards of consistency he deems appropriate.  

The only reason I can think of _not_ to do this would be to maintain consistency with how the Facilities Costume editor works. Which might be appropriate if you decide to implement a Work tab as I've suggested elsewhere; but otherwise, I don't see the point.  

All of the above assumes that it takes more work to disable editing features than to implement them, which would make this a win/win: the player gets more freedom to do what he wants, and you get to slim down your code by removing most of it that's intended to restrict his choices.  On the other hand, if you have to put in code to _enable_ editing functionality, then I could see not wanting to put in the extra effort to implement everything.

**Comment by Perdition-117** [comment]
So, there are two sides to reasons behind these restrictions.

On the gameplay side I made the opinionated decision that it does not make sense for a maid to undergo full plastic surgery for a yotogi session. But I also mostly copied the behaviour from the facility costume setting. Changing hair I think makes sense and is something I have experimented with.

More importantly however, there is the technical side of it, where clothing items is basically a matter of keeping track of a .menu file name, which means the data structure is very simple and uniform. Supporting more exotic slots would require storing various slider values and colors, possibly even with the expectation that AddModSlider parameter would also be stored. All these parameters also need to be carefully restored when leaving the scene. Probably all doable (except maybe the AddModSlider bit), but combined with the subjective side made it an easy opt out at the time.

Opt-in vs opt-out largely does not matter. The actual imposing or non imposing of the restrictions is not a big deal. The technical reasons for them are.
